conference/react-js/src/components/MyInfo/MyInfo.jsx

```diff
@@ -175,72 +207,103 @@ class MyInfo extends Component {
             tabIndex="-1"
           >Change photo</span>
         </div>
-        <form className="my-info" name="" noValidate>
-          <TextareaBlock
-            id="my-info-bio"
-            labelClassName="form-label_required my-info__label
-              my-info__label_bio"
-            className="textarea textarea-focus my-info__bio"
-            label="Short Bio"
-            name="bio"
-            maxLen={2000}
-            value={bio}
-            onChange={this.handleInput}
-          />
-          <InputBlock
-            id="my-job-title"
-            labelClass="form-label_required my-info__label"
-            label="Job Title"
-            name="job"
-            inputClass="field_border my-info__field_job"
-            maxLength={256}
-            value={job}
-            onChange={this.handleInput}
-          />
-          <InputBlock
-            id="my-info-company"
-            labelClass="form-label_required my-info__label"
-            label="Company"
-            name="company"
-            inputClass="field_border my-info__field_company"
-            maxLength={256}
-            value={company}
-            onChange={this.handleInput}
-          />
-          <TextareaBlock
+        <form className="my-info">
+          <div className="input-required input-required__bio">
+            <TextField
+              id="my-info-bio"
+              name="bio"
+              floatingLabelText="Short Bio*"
+              multiLine
+              fullWidth
+              rows={5}
+              maxLength={2000}
+              value={bio || ''}
+              onChange={this.handleInput}
+              onBlur={this.requiredInput}
+              required
+            />
+            <span
+              data-requiredInput="bio"
+              className="my-info__required-message"
+            >
+              * this field is required</span>
+          </div>
+          <div className="input-wrapper">
+            <div className="input-required">
+              <TextField
+                id="my-job-title"
+                name="job"
+                floatingLabelText="Job Title*"
+                fullWidth
+                maxLength={256}
+                value={job || ''}
+                onChange={this.handleInput}
+                onBlur={this.requiredInput}
+                required
+              />
+              <span
+                data-requiredInput="job"
+                className="my-info__required-message"
+              >
+                * this field is required</span>
```

Comment:
I am sure that this span should be under TextField component displayed in case the value is required. it will be much better instead of appending it manually to each TextField

Reply:
I gave them visibility: hidden, because all other content is affected when they are shown
positioning of these spans is not a good idea